neoutils/connection.go

```diff
@@ -50,6 +52,11 @@ func DefaultConnectionConfig() *ConnectionConfig {
 	}
 }
 
+//we need to initialize the logger as dependent services throw an error otherwise
```

Comment:
Let's add something more descriptive that explains why dependent services would throw error: 

> // we need to initialize the logger as this library relies on initialized global variable for the logger object. Othertwise dependant services that don't initialize such global object would get errors using this library.